src/plugins/visualize/public/application/components/visualize_listing.tsx

```diff
@@ -135,16 +136,22 @@ export const VisualizeListing = () => {
   const deleteItems = useCallback(
     async (selectedItems: object[]) => {
       await Promise.all(
-        selectedItems.map((item: any) => savedObjects.client.delete(item.savedObjectType, item.id))
+        selectedItems.map((item: any) => savedVisualizations.delete(item.id))
```

Comment:
the change to savedVisualizations I've left in here since its a strict improvement from the generic saved objects client that existed before.

Reply:
Thanks for the callout - just tested with visbuilder and confirms this fails. I'll revert this part.